addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -3,18 +3,18 @@
     <data>
         <!-- Templates for Dynamic Snippet -->
         <template id="dynamic_filter_template_product_product_add_to_cart" name="Classic Card">
-            <t t-foreach="records" t-as="data">
+            <t t-foreach="records" t-as="data" data-thumb="/website_sale/static/src/img/snippets_options/product_add_to_cart.svg">
```

Comment:
How can this work? A `data-XXX` on a `<t>` instruction... that probably does nothing, right? :thinking:

Reply:
Getting infos from the template is indeed special, see https://github.com/odoo/odoo/blob/a00d53d9fff027f1f2d50687a005f176ea45adcd/addons/website/controllers/main.py#L389

I only built on top of it to add extra data-attributes, such as a thumbnails for the templates inside the option.

Reply:
Grmbl, ok :) I guess there will be more room for improvements in 16.X :)